### PR TITLE
add srwac

### PR DESCRIPTION
**Pull request by IvanZidov**

### CODE REVIEW

**Comment by IvanZidov** [comment]
Connection error failed. Need rerun

File: datasets/srwac/srwac.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                },
+            ),
+        ]
+
+    def _generate_examples(self, filepath):
+        for fp in filepath:
+            with open(fp, encoding="utf8") as f:
+                for id_, row in enumerate(f):
+                    yield id_, {
+                        "sentence": row,
```
Comment (lhoestq) [inline datasets/srwac/srwac.py:90]:
```suggestion
                        "sentence": row.strip(),
```

**Review by lhoestq — APPROVED** [review]
Nice thank you !

**Comment by lhoestq** [comment]
merging since the CI is fixed on master

**Pull request merged**